chart/githubconnector/templates/application.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Values.name }}
```

Comment:
It's better to rely on `{{ .Release.Name }}` Helm makes sure it is unique. Read more at: https://helm.sh/docs/chart_best_practices/#resource-naming-in-templates